assembly/compliance-tool/src/site/markdown/index.md.vm

```diff
@@ -0,0 +1,87 @@
+# How to use the ComplianceTool
+
+This is a frontend of ${docNameCap} that can be used as a compliance manager tool.
+This is especially targeted to create clean and approved metadata in a SW360 instance.
+
+This tool can be used for a round trip in order to enrich and clear metadata in a SW360 instance. 
+
+#[[##]]# Configuring
+
+#[[##]]# Running
+
+The Compliance Tool is executed from the command line with this simple line:
+
+```
+java -jar path\to\complianceTool.jar <complianceMode> path\to\configuration.properties
+```
+
+- Jar file: This is the actual compliance tool jar. Execute it with a `java -jar` command.
+- The `complianceMode` can be one of two modes: `exporter` for the [SW360 Exporter](#SW360Exporter)
+or `updater` for the [SW360 Updater](#SW360Updater).
+- Properties file: both modes require a file containing properties like paths and credentials.
+
+#[[##]]# <a name="SW360Exporter">SW360 Exporter</a>
+
+The SW360 Exporter requests all components from a given SW360 instance
+and filters the releases of the components to check if they have a state
+that shows them as "cleared" or "approved".
+It then creates a list with the data of those releases and writes them
+sorted by creation date (from latest to oldest) in a csv file.
```

Comment:
From this paragraph it is not fully clear to me, which kind of releases are written to the csv file: the cleared/approved ones or the other ones. I am a bit afraid that somebody who does not know the background of the tool will not be able to understand this description.

As a suggestion: Could there be a (short) description of the intended round-trip at the top of the page, so that the basic workflow becomes clearer? Something like
* Data about incomplete components/releases is exported from SW360 to csv
* The csv file is then edited manually
* Based on the edits in the csv file, SW360 is updated